frontend/src/static/js/v_authorship.js

```diff
@@ -88,18 +89,36 @@ window.vAuthorship = {
 
     getRepoProps(repo) {
       if (repo) {
-        const author = repo.users.filter((user) => user.name === this.info.author);
-        if (author.length > 0) {
-          this.info.name = author[0].displayName;
-          this.filesLinesObj = author[0].fileTypeContribution;
+        if (this.info.isMergeGroup) {
+          // sum of all users' file type contribution
+          repo.users.forEach((author) => {
+            this.updateTotalFileTypeContribution(author.fileTypeContribution);
+          });
+        } else {
+          const author = repo.users.filter((user) => user.name === this.info.author);
```

Comment:
I know it's not part of this PR, but can consider using [find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here

Reply:
Done